### PR TITLE
AuditedBaseEntity now inherits from BaseEntity (#1137)

### PR DESCRIPTION
**Pull request by aswarthm**
-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Documentation is added/updated where necessary
-   [ ] Tests are added where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

As discussed in issue #1137, I have changed the inheritance class of AuditedBaseEntity from IAuditedEntityBase to BaseEntity

### CODE REVIEW

**Comment by aswarthm** [comment]
Checks are failing because BaseEntity takes in a parameter "TKey", but AuditedBaseEntity isn't passed with any parameters. How do I fix this

File: generators/server/templates/dotnetcore/src/Project.Domain/Entities/AuditedEntityBase.cs.ejs [hunk 1]
```diff
@@ -17,7 +17,7 @@ using <%= namespace %>.Domain.Entities.Interfaces;
 
 namespace <%= namespace %>.Domain.Entities;
 
-public abstract class AuditedEntityBase : IAuditedEntityBase
+public abstract class AuditedEntityBase : BaseEntity
```
Comment (nicolas63) [inline generators/server/templates/dotnetcore/src/Project.Domain/Entities/AuditedEntityBase.cs.ejs:20]: You need to do something like this

public abstract class AuditedEntityBase<TKey : BaseEntity<TKey>, IAuditedEntityBase

**Review by nicolas63 — CHANGES_REQUESTED** [review]

File: generators/server/templates/dotnetcore/src/Project.Domain/Entities/AuditedEntityBase.cs.ejs (hunk 1 above)
Comment (aswarthm) [reply generators/server/templates/dotnetcore/src/Project.Domain/Entities/AuditedEntityBase.cs.ejs]: > like this sorry i have write the previous code from my phone:
> 
> ```
> public abstract class AuditedEntityBase<TKey> : BaseEntity<TKey>, IAuditedEntityBase
> ```

sorry I'm new to this, Thanks for the help

**Review by nicolas63 — APPROVED** [review]

**Comment by nicolas63** [comment]
looks good to me, thanks for your contribution

**Pull request merged**